pipeline/02-assess.R

```diff
@@ -154,7 +183,8 @@ assessment_data_merged %>%
   select(
     meta_year, meta_pin, meta_class, meta_card_num, meta_lline_num,
     meta_modeling_group, ends_with("_num_sale"), pred_card_initial_fmv,
-    all_of(params$model$predictor$all), township_code
+    all_of(params$model$predictor$all),
+    meta_strata_is_imputed, township_code
```

Comment:
**suggestion:** Let's change the prefix of this variable to `flag_` since its purpose fits slightly better with that group of vars.